gap/attr.gi

```diff
@@ -8,6 +8,85 @@
 #############################################################################
 ##
 
+# The next method is (yet another) DFS as described in
+# http://www.eecs.wsu.edu/~holder/courses/CptS223/spr08/slides/graphapps.pdf
+
+InstallMethod(ArticulationPoints, "for a digraph", [IsDigraph],
+function(digraph)
+  local copy, nbs, counter, visited, num, low, parent, points, v_stack,
+  w_stack, depth, 1st_root_nb, v, w, i;
+
+  if HasIsConnectedDigraph(digraph) and not IsConnectedDigraph(digraph) then
+    return [];
+  elif DigraphNrVertices(digraph) < 1 then
```

Comment:
Minor point: this `<` could be replaced by `<=` to avoid a tiny little bit more work.